ui/utilities/media-objects/flavors/base/_index.scss

```diff
@@ -62,7 +62,7 @@
     }
 
     &.slds-media--small .slds-media__figure {
-      margin-left: $spacing-xx-small;
+      margin-#{$left}: $spacing-xx-small;
```

Comment:
This will make impossible to share the same css on a site that supports both.

A common use case is to have a site that is ltr by default but can be toggled to `rtl` for users on a specific region or based on a config.

```html
<body className={isRtl ? 'slds-rtl' : 'slds-ltr'}>
   ... `-end` class modifier with mean `left` only if the parent is 'slds-rtl' 
   ... `-start` class modifier with mean `right` only if the parent is 'slds-rtl'
</body>
```